pkg/webui/console/containers/lora-cloud-das-form/index.js

```diff
@@ -95,6 +95,7 @@ const defaultValues = {
   data: {
     token: '',
     server_url: LORA_CLOUD_DAS.DEFAULT_SERVER_URL,
+    f_port_set: [192, 197, 198, 199],
```

Comment:
I think that in the frontend we prefer that this becomes something like `LORA_CLOUD_DAS.DEFAULT_PORT_SET`.